osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -0,0 +1,356 @@
+                                Padding = new MarginPadding { Left = 20 },
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Vertical
+                            },
+                            deletedChildrenPlaceholder = new DeletedChildrenPlaceholder
+                            {
+                                ShowDeleted = { BindTarget = ShowDeleted }
+                            }
+                        }
+                    }
+                }
+            };
+
+            deletedChildrenPlaceholder.DeletedCount.Value = comment.GetDeletedChildrenCount;
+
+            if (comment.UserId.HasValue)
+                username.AddUserLink(comment.User);
+            else
+                username.AddText(comment.LegacyName);
+
+            if (comment.EditedAt.HasValue)
+            {
+                info.Add(new SpriteText
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Font = OsuFont.GetFont(size: 12),
+                    Text = $@"edited {HumanizerUtils.Humanize(comment.EditedAt.Value)} by {comment.EditedUser.Username}"
+                });
+            }
+
+            if (!comment.IsDeleted)
+            {
+                var formattedSource = MessageFormatter.FormatText(comment.GetMessage());
+                message.AddLinks(formattedSource.Text, formattedSource.Links);
+            }
+            else
+            {
+                content.FadeColour(OsuColour.Gray(0.5f));
+                votePill.Hide();
+            }
+
+            if (comment.IsTopLevel)
+            {
+                AddInternal(new Container
+                {
+                    RelativeSizeAxes = Axes.X,
+                    Height = 1.5f,
+                    Anchor = Anchor.BottomCentre,
+                    Origin = Anchor.BottomCentre,
+                    Child = new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = OsuColour.Gray(0.1f)
+                    }
+                });
+
+                if (comment.ChildComments.Any())
+                {
+                    AddInternal(new ChevronButton(comment)
+                    {
+                        Anchor = Anchor.TopRight,
+                        Origin = Anchor.TopRight,
+                        Margin = new MarginPadding { Right = 30, Top = margin },
+                        Expanded = { BindTarget = childrenExpanded }
+                    });
+                }
+            }
+
+            comment.ChildComments.ForEach(c => childCommentsContainer.Add(new DrawableComment(c)
+            {
+                ShowDeleted = { BindTarget = ShowDeleted }
+            }));
+        }
+
+        protected override void LoadComplete()
+        {
+            ShowDeleted.BindValueChanged(onShowDeletedChanged, true);
```

Comment:
Do it inline? `BindValueChanged(show => this.FadeTo(show.NewValue ? 1 : 0), true)`